### PR TITLE
[fix] `order`: recognize ".." as a "parent" path

### LINKED ISSUE #1405: order: '..' is not recognized
https://github.com/benmosher/eslint-plugin-import/pull/1375 added support for "unknown" imports. Imports from `..` are classified as *unknown*. I think it ought to be *parent*.

```js
import foo from '../foo'
import { bar } from '..'
import baz from '.'
```

### PR DESCRIPTION
**Pull request by golopot**
fixes #1405

### CODE REVIEW

File: tests/src/rules/order.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { test, getTSParsers } from '../utils'
 import { RuleTester } from 'eslint'
 
 const ruleTester = new RuleTester()
-    , rule = require('rules/order')
+    , rule = require('../../../src/rules/order')
```
Comment (ljharb) [inline tests/src/rules/order.js]: not sure why this is here; `npm run mocha path/to/file` will set up the NODE_PATH for you to point to `src`.
```suggestion
    , rule = require('rules/order')
```

Comment (golopot) [reply tests/src/rules/order.js]: This is how I run tests:
```sh
npx mocha --compilers js:babel-register tests/src/rules/order.js
```
Without that change the `require` fails.

**Comment by golopot** [comment]
> Per #1405 (comment), should this go through the resolver to consider "parent"-ness?

I believe it is better to order the import statements by the apparent path rather the resolved path. What I wanted for this rule is a consistent ordering that looks nice. Ordering by resolved path does not provide a consistent ordering that looks nice.

File: tests/src/rules/order.js (hunk 1 above)
Comment (ljharb) [reply tests/src/rules/order.js]: Right, that's not a valid way to run tests in this repo - you need to use `npm run mocha path/to/file` instead.

**Comment by ljharb** [comment]
True, that makes sense.

**Pull request merged**